cmd/lakefs/cmd/run.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/go-co-op/gocron"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"github.com/treeverse/lakefs/modules/block/factory"
```

Comment:
🎉

Reply:
One thing I forgot! 
Let's explicitly give it a name to avoid collisions in future factories:
```suggestion
	blockfactory "github.com/treeverse/lakefs/modules/block/factory"
```